tests/snuba/api/endpoints/test_organization_events_ourlogs.py

```diff
@@ -0,0 +1,88 @@
+from datetime import datetime, timezone
+
+import pytest
+
+from tests.snuba.api.endpoints.test_organization_events import OrganizationEventsEndpointTestBase
+
+
+class OrganizationEventsOurLogsEndpointTest(OrganizationEventsEndpointTestBase):
+    dataset = "ourlogs"
+
+    def do_request(self, query, features=None, **kwargs):
+        query["useRpc"] = "1"
+        return super().do_request(query, features, **kwargs)
+
+    def setUp(self):
+        super().setUp()
+        self.features = {
+            "organizations:ourlogs-enabled": True,
+        }
+
+    @pytest.mark.querybuilder
```

Comment:
note to self, probably should add a mark for searchresolver instead of just querybuilder